sources/packages/backend/libs/integrations/src/esdc-integration/disbursement-receipt-integration/disbursement-receipt.processing.service.ts

```diff
@@ -141,6 +150,51 @@ export class DisbursementReceiptProcessingService {
       }
     }
     result.processSummary.push(`Processing file ${remoteFilePath} completed.`);
+    const batchRunDate = responseData.header.batchRunDate;
+    result.processSummary.push(
+      `Processing provincial daily disbursement CSV file which are not sent on ${getISODateOnlyString(
+        batchRunDate,
+      )}.`,
+    );
+
+    // Create a CSV file of the total records sent on the process date and
+    // email the content to the ministry users.
+    try {
+      // Populate the reportName and batchRunDate to the reportsFilterMode.
+      const reportFilterModel: ReportsFilterModel = {
+        reportName: DAILY_DISBURSEMENT_REPORT_NAME,
+        params: { batchRunDate },
```

Comment:
Is it not the case of using "File date" + "Sequence number" for report generation as per the technical AC?